sphinx/tutorial/rose/furthertopics/macro-development.rst

```diff
@@ -90,16 +90,20 @@ Open ``planet.py`` in a text editor and paste in the following code:
 
 .. code-block:: python
 
-   #!/usr/bin/env python2
-   # -*- coding: utf-8 -*-
+   #!/usr/bin/env python3
 
    import re
    import subprocess
 
-   import metomi.rose.macro
+   try:
+       from metomi.rose.macro import MacroBase
+       ROSEV = 2
+   except ImportError:
+       from rose.macro import MacroBase
+       ROSEV = 1
```

Comment:
ROSEV might be a little cryptic, `PY_3 = True` might be a little clearer.

Reply:
Ay. Done